website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -396,15 +396,19 @@ spec:
                       provided permissions in this intention will be subject to the default
                       intention behavior is defined by the default [ACL policy](/docs/agent/options#acl_default_policy).<br><br>
                       This should be omitted for an L4 intention as it is mutually exclusive with
-                      the \`Action\` field.`,
+                      the \`Action\` field.<br><br>
+                      Setting \`Permissions\` is not valid if a wildcard is used for the \`Name\` or \`Namespace\` since they can only be
```

Comment:
```suggestion
                      Setting \`Permissions\` is not valid if a wildcard is used for the \`Name\` or \`Namespace\` because they can only be
```
This is a little grammar nitpick, but a common diction error is using "since" when you mean "because". The "since" preposition relates to time, whereas "because" relates to function. Have some salt with this comment.

Reply:
Nice tip thanks!